structured/structured.proto

```diff
@@ -102,6 +106,7 @@ message TableDescriptor {
 // Permissions are applied to all tables in the namespace.
 message DatabaseDescriptor {
   optional string name = 1 [(gogoproto.nullable) = false];
-  optional uint32 id = 2 [(gogoproto.nullable) = false, (gogoproto.customname) = "ID"];
+  optional uint32 id = 2 [(gogoproto.nullable) = false,
+      (gogoproto.customname) = "ID", (gogoproto.casttype) = "ID"];
```

Comment:
ah. cool. Then please at a todo around GetID to remove them when no longer needed.